src/pages/taesooKim/Main/CommentList.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from "react";
+import Comment from "./Comment";
+
+export class CommentList extends Component {
+  render() {
+    const { comment, deleteComment } = this.props;
+    return (
+      <div>
+        {comment.map(comment => {
```

Comment:
```suggestion
        {comments.map(comment => {
```
변수명을 복수형으로 지어주면 변수명만 보고도 배열 형태라는 걸 쉽게 추론할 수 있을듯 하네요

Reply:
네네 수정하였습니다🐸